invenio_access/__init__.py

```diff
@@ -142,6 +142,47 @@
 >>> admin_identity.can(permission_index)
 True
 
+How to discover existing actions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Modules that intend to implement a given set of actions can register them in
+entry points in the corresponding `setup.py`, e.g.
+
+```python
+entry_points={
+    'invenio_access.actions': [
+        'records_read_all'
+        ' = invenio_records.permissions:records_read_all',
+        'records_create_all'
+        ' = invenio_records.permissions:records_create_all',
+        'records_update_all'
+        ' = invenio_records.permissions:records_update_all',
+        'records_delete_all'
+        ' = invenio_records.permissions:records_delete_all',
+    ],
+}
+```
+
+In order to discover which actions are available in a given installation one
+can retrieve them via:
+
+>>> app.extensions['invenio-access'].actions
+{'admin-access': Need(method='action', value='admin-access'),
+ 'files-rest-bucket-create': Need(method='action', value='files-rest-bucket-create'),
```

Comment:
Sure: this is just an example. In fact I am writing: "In order to discover which actions are available".
How would you suggest to improve it?